fxa-js-client tests failing locally

Blocks #693.

Not sure if anyone else sees this or my installation is screwy but fwiw:

```
~/c/f/p/fxa-js-client (pb/693-new-entrypoint-params) $ npm t

> fxa-js-client@1.0.8 test /Users/philbooth/code/fxa/packages/fxa-js-client
> grunt test

Running "intern:node" (intern) task
Running with mocks...
FATAL ERROR
TypeError: Cannot read property 'hex' of undefined
  at <client/lib/credentials.js:12:30>
  at runFactory  <node_modules/dojo/dojo.js:1104:43>
  at execModule  <node_modules/dojo/dojo.js:1232:5>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at <node_modules/dojo/dojo.js:1267:6>
  at guardCheckComplete  <node_modules/dojo/dojo.js:1247:5>
Warning: FATAL ERROR
TypeError: Cannot read property 'hex' of undefined
  at <client/lib/credentials.js:12:30>
  at runFactory  <node_modules/dojo/dojo.js:1104:43>
  at execModule  <node_modules/dojo/dojo.js:1232:5>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at execModule  <node_modules/dojo/dojo.js:1223:12>
  at <node_modules/dojo/dojo.js:1267:6>
  at guardCheckComplete  <node_modules/dojo/dojo.js:1247:5> Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
```

--- a/.circleci/test-js-client.sh
+++ b/.circleci/test-js-client.sh
@@ -6,8 +6,7 @@ DIR=$(dirname "$0")
 if grep -e "$MODULE" -e 'all' $DIR/../packages/test.list; then
   sudo apt-get install -y default-jre
   # for some reason js-client tests fail if we cache node_modules
-  npm install
-  npx grunt sjcl
+  npm ci
   node_modules/.bin/grunt eslint
   npm test
   # TODO ./tests/ci/travis-auth-server-test.sh

--- a/packages/fxa-js-client/CONTRIBUTING.md
+++ b/packages/fxa-js-client/CONTRIBUTING.md
@@ -40,7 +40,7 @@ This project is production Mozilla code and subject to our [engineering practice
 Note: Java is required to build the library due to a custom SJCL build.
 
 ```
-npm run-script setup
+npm ci
 npm start
 ```
 

--- a/packages/fxa-js-client/package.json
+++ b/packages/fxa-js-client/package.json
@@ -8,7 +8,7 @@
     "start": "grunt",
     "test": "grunt test",
     "test-local": "intern-client config=tests/intern auth_server=LOCAL",
-    "setup": "npm install && grunt sjcl",
+    "postinstall": "grunt sjcl",
     "contributors": "git shortlog -s | cut -c8- | sort -f > CONTRIBUTORS.md"
   },
   "main": "node/index.js",

